importer/api/__init__.py

```diff
@@ -84,6 +84,9 @@ def seekable():
         default_storage.delete(filename)
         rmtree(tempdir)
 
+    import_course_from_file.seekable = seekable
+    return import_course_from_file
```

Comment:
Now the function is attaching a method to itself and returning itself. This is fraught with potential problems. It changes the function on import vs run. You're also returning something completely unrelated to the purpose of the function